modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1363,6 +1363,152 @@ definition {
 			actionLabel = "Action Label");
 	}
 
+	@description = "Verify that power users should be able to manage standalone action permissions in object definition permissions"
+	@priority = 5
+	test CanManageStandalonePermissionsAtObjectDefinition {
```

Comment:
Add property `property portal.acceptance = "true";` to tests with `@priority = 5`

Reply:
Right @andreefarias, I'll change, thanks